api/app/Http/Controllers/OccurrenceController.php

```diff
@@ -600,7 +600,7 @@ public function skeletalImport(Request $request){
 	}
 
 	/**
-	 * @off_OA\Get(
+	 * @OA\Get(
```

Comment:
@egbot I had to change this in order to get `php artisan swagger-lume:generate` to run successfully.